gui/styles/_wizard.styl

```diff
@@ -81,27 +84,83 @@ a.more-info
   color var(--slateGrey)
 
 
-.folder__selector
-  @extend $form-select select
-  width: 100%
-  font-size: 1.1em
-  line-height: 1em
-  margin 0
-  text-align left
-  text-decoration: none
+.folder__subtitle
+  margin-bottom 0
+
+.folder__config
+  margin-top 1rem
+
+  ul
+    margin-bottom 0
+    padding-inline-start: 1rem;
+
+  li + li
+    margin-top 1rem
+
+.folder__config-option__title
+  font-weight bold
+
+.folder__config-option__button
+  margin-left .5rem
+  color var(--dodgerBlue)
+
+.folder__path
+  position relative
+  padding-left 1.1rem
+
+  font-weight bold
+  font-style italic
+
+  &:before
+    @extend $icon
+    display: inline-block
+    vertical-align: middle
+    -webkit-mask-image: embedurl('./node_modules/cozy-ui/assets/icons/ui/folder.svg')
+    -webkit-mask-position: center
+    -webkit-mask-size: cover
```

Comment:
You don't have a tool for putting the prefixed version of CSS properties for you (like autoprefixer)?

Reply:
No, Desktop doesn't have any bundler so no `autoprefixer`.